src/components/ui/interactive-keyboard.tsx

```
@@ -213,166 +306,166 @@ export function InteractiveKeyboard({ className, onKeyClick, ...props }: Interac
-          <button type="button" className="ikb-btn-4" onClick={() => handleKeyClick("return")}><span className="ikb-lr ikb-xs">return</span></button>
+          <button type="button" className="ikb-btn-2" {...keyProps("a")}><span>A</span></button>
+          <button type="button" className="ikb-btn-2" {...keyProps("s")}><span>S</span></button>
+          <button type="button" className="ikb-btn-2" {...keyProps("d")}><span>D</span></button>
+          <button type="button" className="ikb-btn-2" {...keyProps("f")}><span>F</span><span className="ikb-bump"></span></button>
+          <button type="button" className="ikb-btn-2" {...keyProps("g")}><span>G</span></button>
+          <button type="button" className="ikb-btn-2" {...keyProps("h")}><span>H</span></button>
+          <button type="button" className="ikb-btn-2" {...keyProps("j")}><span>J</span><span className="ikb-bump"></span></button>
+          <button type="button" className="ikb-btn-2" {...keyProps("k")}><span>K</span></button>
+          <button type="button" className="ikb-btn-2" {...keyProps("l")}><span>L</span></button>
+          <button type="button" className="ikb-btn-2" {...keyProps(";")}><span className="ikb-sm">:<br/>;</span></button>
+          <button type="button" className="ikb-btn-2" {...keyProps("'")}><span className="ikb-sm">&quot;<br/>'</span></button>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)